src/azure-cli/azure/cli/command_modules/appservice/_params.py

```diff
@@ -229,6 +229,9 @@ def load_arguments(self, _):
         with self.argument_context(scope + ' config ssl import') as c:
             c.argument('key_vault', help='The name or resource ID of the Key Vault')
             c.argument('key_vault_certificate_name', help='The name of the certificate in Key Vault')
+        with self.argument_context(scope + ' config ssl sync') as c:
+            c.argument('certificate_name', help='The name of the certificate to be synced')
+            c.argument('key_vault', help='The keyvault backing this cert, where the data will be synced from')
```

Comment:
What is the supported format? Name? ID?

Reply:
KeyVault can be both name and ID, I added some notes to clarify that